thoth/user_api/api_v1.py

```diff
@@ -574,21 +575,27 @@ def list_buildlogs(page: int = 0):
 def get_python_package_versions_count():
     """Retrieve number of Python package versions in Thoth Knowledge Graph."""
     from .openapi_server import GRAPH
-    return GRAPH.get_python_package_versions_count_all(
-    )
+    return {"count": GRAPH.get_python_package_versions_count_all(
+        distinct=True
+    )}
 
 
 def get_python_package_versions_count_per_package_name(page: int = 0):
     """Retrieve number of Python package versions per package name in Thoth Knowledge Graph."""
     from .openapi_server import GRAPH
-    return (
-        GRAPH.get_python_package_versions_all_count(
+    output = GRAPH.get_python_package_versions_all_count(
             count=PAGINATION_SIZE,
-            start_offset=page*PAGINATION_SIZE),
+            start_offset=page*PAGINATION_SIZE,
+            distinct=True,
+            sort_by=QuerySortTypeEnum.PACKAGE_NAME)
+    total_pages = int(output.count/PAGINATION_SIZE)
```

Comment:
501 records will result in 51 pages, the last one will have just one record:

```python
import math
math.ceil(output.count / PAGINATION_SIZE)
```

Reply:
shouldn't be `math.floor`? if 50 records -> 0 pages because we have page 0.

Reply:
The number of all pages is then 1, the very first page has 50 records and has index 0.

If there would be 0 pages, then there are no records.